docs/gds-supported-platform/gsp-service-operator.md

```diff
@@ -210,6 +229,49 @@ postgres=>
 
 You could also get the read endpoint using the ReadEndpoint key.
 
+## How to connect to a created Redis
+
+If you make a pod like the one above:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: alexs-test-pod
+spec:
+  containers:
+  - name: myapp-container
+    image: govsvc/redis-and-stunnel
+    command: ['sleep', '1000000']
+    env:
+    - name: STUNNEL_HOSTNAME
+      valueFrom:
+        secretKeyRef:
+          name: alexs-test-cluster-secret
+          key: ClusterPrimaryRedisHostname
+    - name: STUNNEL_PORT
+      valueFrom:
+        secretKeyRef:
+          name: alexs-test-cluster-secret
+          key: ClusterPrimaryRedisPort
+    - name: REDISCLI_AUTH
+      valueFrom:
+        secretKeyRef:
+          name: alexs-test-cluster-secret
+          key: SecretAuthToken
+```
+
+You will be able to exec into this pod and connect to Redis after configuring and running stunnel:
+```
+$ gds sandbox kubectl exec -n sandbox-gsp-service-operator-test alexs-test-pod -c myapp-container -it /bin/bash
+root@alexs-test-pod:/data# echo "  connect = $STUNNEL_HOSTNAME:$STUNNEL_PORT" >> /etc/stunnel/redis-cli.conf
+root@alexs-test-pod:/data# stunnel /etc/stunnel/redis-cli.conf
+root@alexs-test-pod:/data# redis-cli -h 127.0.0.1
```

Comment:
omg i can't believe redis-cli doesn't support TLS 😨

Reply:
🤔 I wonder if there's a clever way we could get istio to do this bit for us? so it looks to our apps like they're talking to redis over http, but envoy magically handles the TLS bit? anyway that's not a blocker for this PR

Reply:
I know right?

Reply:
Not sure re Istio.